Scripts/symbol_table_i386.py

```diff
@@ -0,0 +1,70 @@
+from os import popen, system
+import sys
+
+PARSED =  "parsed_i386.sym"
+KERNEL =  sys.argv[1]
+system(f"touch {PARSED}")
+writer = open(PARSED, "w")
```

Comment:
```suggestion
writer = open(PARSED, "w+")
```

What happens if the file already exists? wont it potentially cause conflicts.

Reply:
No it just overwrites the old contents.
I ran it over and over and over without a problem

Reply:
What if the old content is longer than the new content

Reply:
hello world

file.write('bye')

byelo world

Reply:
Oh yeah you right.
My python skills are rusty, I thought write() cleared the file unless you seek() the end of it and append text